torch/csrc/jit/passes/xnnpack_rewrite.cpp

```diff
@@ -297,11 +297,13 @@ void FoldPrePackingOps(script::Module& m) {
 }
 
 void optimizeForMobile(script::Module& m) {
-  m.eval();
-  m = FoldConvBatchNorm2d(m);
-  insertPrePackedOps(m);
-  m = freeze_module(m);
-  FoldPrePackingOps(m);
+  auto moduleClone = module.clone();
```

Comment:
Do variable names follow camel case? From what I noticed no. If so change accordingly. Otherwise look great.

Reply:
good point, seems like not.